README.md

```diff
@@ -543,7 +543,7 @@ is_mistral_derived_model:
 is_qwen_derived_model:
 
 # optional overrides to the base model configuration
-model_config:
+model_config_overrides:
```

Comment:
I just noticed this, but we would need to deprecate the old name (add valueerror)

Reply:
we can't actually deprecate it with Pydantic because `model_config` is an internal variable name for pydantic models